docs/reference/kubeedge-api/operations-resources/image-pre-pull-job-v1alpha1.md

```diff
@@ -0,0 +1,800 @@
+---
```

Comment:
operations still have another API: nodeUpgradeJob  Ref: https://github.com/kubeedge/kubeedge/blob/master/pkg/apis/operations/v1alpha1/type.go

Reply:
we added this API(nodeUpgradeJob):https://github.com/greensong/website/blob/master/docs/reference/kubeedge-api/operations-resources/node-upgrade-job-v1alpha1.md